app/controllers/internal/feedback_messages_controller.rb

```diff
@@ -5,13 +5,13 @@ def index
     @q = FeedbackMessage.includes(:reporter, :offender, :affected).
       order(created_at: :desc).
       ransack(params[:q])
-    @feedback_messages = @q.result.page(params[:page] || 1).per(5)
+    @feedback_messages = @q.result.limit(5).page(params[:page] || 1).per(5)
```

Comment:
I think this is already paginating this right? Unless you are changing the page behaviour to only show 5 feedback messages.

Reply:
Yeah not sure if this is redundant? Worst case scenario though is that it doesn't do anything, which is not bad at all.

Reply:
Ah right it is doing the same thing. Thanks!
```suggestion
    @feedback_messages = @q.result.page(params[:page] || 1).per(5)
```